sites/friday/src/get-started/here/configure/_index.md

```diff
@@ -0,0 +1,108 @@
+
+```yaml {location=".upsun/config.yaml"}
+{{< code-link destination="/create-apps/app-reference.html" text="applications:" title="Top-level key for all applications" >}}
+    # Configuration for all applications within an environment.
+    {{< code-link destination="/create-apps/app-reference.html" text="myapp:" title="Configuration for a unique application" >}}
+        # configuration for the application 'myapp'
+
+{{< code-link destination="/add-services.html" text="services:" title="Top-level key for all services" >}}
+    # Configuration for all services within an environment.
+    {{< code-link destination="/add-services.html" text="db:" title="Configuration for a unique service" >}}
+        # configuration for service 'db'
+
+{{< code-link destination="/define-routes.html" text="routes:" title="Top-level key for all routes, configuring how requests are handled by the Router" >}}
+    # Configuration for routing to applications
+    {{< code-link destination="/define-routes.html" text="'https://{default}/':" title="Configuration for a unique service" >}}
+```
+{{% note %}}
+If you want more information on the structure of this config file, please see dedicated [YAML doc page](/learn/overview/yaml/_index.md)
+{{% /note %}}
+
+Commit your new files:
+
+```bash {location="Terminal"}
+git add . && git commit -m "Add configuration for {{% vendor/name %}}."
+```
+
+and push up to your Upsun project:
+
+```bash
+upsun push
+```
+
+{{% vendor/name %}} will now read your configuration files, begin building your application image and allocate resources to your various containers.
+
+{{< note >}}
+By default, {{% vendor/name %}} will use the default resource allocation for each of your services/apps. To change allocated resources to each of your services/apps, please see [how to set resources to your project](/get-started/here/set-resources) section
+{{< /note >}}
+
+Et voilà, your application is live!
+
+## Next steps
+
+The `project:init` CLI command will use your responses and some framework-detection logic to attempt to set some sane configuration for your project. 
+Your project - like most projects - is unique, and this goal of a seamless first deployment may yet be incomplete at this stage.
+
+Have no fear! {{% vendor/name %}} provides a highly customizable configuration schema that can be adjusted for your particular package manager, environment variables, builds, deploys, workers, crons, and other requirements to get you full deployed.
+
+The best place to start to start exploring what might still be needed are the language-specific configuration pages:
+
+- [JavaScript/Node.js](/get-started/here/configure/nodejs)
+- [PHP](/get-started/here/configure/php)
+- [Python](/get-started/here/configure/python)
+
+With your deployment finished, you may have noticed that the resources allocated to the application and service containers of your project will have been set automatically, and perhaps rather conservatively:
```

Comment:
i think we should drop "and perhaps rather conservatively:". three containers at the default settings is going to run around 190 euro/month so not necessarily "conservative".